_pages/algorithm-development/your-first-algo.md

```diff
@@ -13,224 +13,215 @@ redirect_from:
 
 {% highlight python %}
 if 'url' in input:
   text = Algorithmia.algo('util/Html2Text/0.1.3').pipe(input['url']).result
 {% endhighlight %}
 
-This example shows an algorithm that checks the type of input passed in, and if it is a URL, will call into the Html2Text algorithm. The original caller of your algorithm will be charged for both the first algorithm call as well as the internal algorithm call. The algorithm will automatically use the credentials of the person calling your algorithm.
+This example shows an algorithm that checks the type of input passed in, and if it is a URL, calls the Html2Text algorithm to pull down code from the webpage. Compute associated with both the original algorithm call and the internal algorithm call is tracked and attributed to the original caller's user account.
 
-## (Optional) Delete your Algorithm
+## (Optional) Deleting your Algorithm
 
-If your algorithm has never been published, or has only been published for Private use, you have the option to delete it when it is no longer needed. To do so, go to your algorithm's page and click on the **Manage** tab, then find the **Delete** button at the bottom.
+If your algorithm has never been published, or has only been published for private use, you have the option to delete it when it is no longer needed. To do so, go to your algorithm's page, click on the **Settings** tab, and find the red **Delete** button at the bottom.
 
 <img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/algo_dev_lang/delete_algorithm.png" alt="Delete algorithm" class="screenshot img-sm">
 
-If you have ever published your Algorithm with "Public" selected, it cannot be deleted. This is to ensure that others' code will not break if they have written algorithms or client code which depends on your algorithm.
+If you have ever published your Algorithm with "Public" selected, it cannot be deleted. This is to ensure that others' code will not break if they have written algorithms or client code that depends on your algorithm.
 
 ## Wrapping Up
 
-Now that you've published your first simple algorithm on the platform, you can feel empowered to add even more algorithms. You can write them from scratch, adopt and modify open source algorithms, or even chain multiple pre-existing algorithms together to make a micro service you can call with the Algorithmia API! Get inspired by checking out our <a href="{{site.baseurl}}/tutorials/recipes">Recipes</a>.
+Now that you've published your first simple algorithm on the platform, you are empowered to develop more complex and useful algorithms. You can write them from scratch, adopt and modify open-source algorithms, or even chain multiple pre-existing algorithms together to make a micro service that you can call with the Algorithmia API! Get inspired by checking out our <a href="{{site.baseurl}}/tutorials/recipes">Recipes</a>.
```

Comment:
```suggestion
Now that you've published your first simple algorithm on the platform, you are empowered to develop more complex and useful algorithms. You can write them from scratch, adopt and modify open-source algorithms, or even chain multiple pre-existing algorithms together to make a micro service that you can call with the Algorithmia API! 
```

Reply:
Recipes section is slated for removal - if time allows I'd like the last section mention something like "chain together algorithms to take advantage of our microservice architecture" or something like that - maybe even mentioning use case like data pre-processing algo combined with image processing or something like that to showcase pipelining

Reply:
Yeah that sounds like a great use case / demo to build. I have added it to my personal TODO board, so I think we can consider it resolved here.